src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrentSettings.cs

```diff
@@ -63,6 +65,15 @@ public QBittorrentSettings()
         [FieldDefinition(10, Label = "Initial State", Type = FieldType.Select, SelectOptions = typeof(QBittorrentState), HelpText = "Initial state for torrents added to qBittorrent")]
         public int InitialState { get; set; }
 
+        [FieldDefinition(11, Label = "Remove stalled downloads", Type = FieldType.Checkbox, Advanced = true, HelpText = "Remove stalled downloads from the client. BEWARE: This could get you banned from trackers!")]
+        public bool RemoveStalledDownloads { get; set; }
+
+        [FieldDefinition(12, Label = "Stalled age threshold", Type = FieldType.Textbox, Advanced = true, HelpText = "Age threshold before a stalled torrent is considered for deletion (in seconds)")]
```

Comment:
`FieldDefinition` has a `Units` Prop which will show right aligned placeholder in the UI for the units. Lets use that and remove from `HelpText`